contracts/truefi2/TrueRateAdjuster.sol

```diff
@@ -121,6 +127,24 @@ contract TrueRateAdjuster is ITrueRateAdjuster, UpgradeableClaimable {
         borrowLimitConfig = BorrowLimitConfig(40, 7500, 1500, 1500);
     }
 
+    /**
+     * @dev Set whether `pool` is included in TVL calculation
+     */
+    function setTVLPool(ITrueFiPool2 pool, bool isIncluded) external onlyOwner {
+        if (isIncluded) {
+            tvlPools.push(pool);
+        } else {
+            for (uint256 i = 0; i < tvlPools.length; i++) {
+                if (tvlPools[i] == pool) {
+                    tvlPools[i] = tvlPools[tvlPools.length - 1];
+                    tvlPools.pop();
+                    break;
+                }
+            }
+        }
+        emit TVLPool(pool, isIncluded);
+    }
```

Comment:
I think `isIncluded` name is not clear enough. Maybe we want to split this into 2 functions?
Also we should probably add check preventing adding same pool twice

Reply:
These are good points, thanks! I'll make these changes.